client/me/help/help-contact/index.jsx

```diff
@@ -64,6 +65,10 @@ const HelpContact = React.createClass( {
 			this.props.connectHappychat();
 		}
 
+		if ( config.isEnabled( 'help/directly' ) ) {
+			this.props.initializeDirectly();
```

Comment:
Quick ping @mtias — are you able to elaborate on what you mean about async loading in this context?